src/heaplayers/heaps/top/mallocheap.h

```diff
@@ -60,7 +60,7 @@ namespace HL
 	  void *p =::malloc (sz + cubmem::MMON_METAINFO_SIZE);
 	  if (p != NULL)
 	    {
-	      mmon_add_stat ((char *) p, sz + cubmem::MMON_METAINFO_SIZE, __FILE__, __LINE__);
```

Comment:
This is already fixed in https://github.com/CUBRID/cubrid/pull/5188.

Reply:
Done.